README.md

```diff
@@ -6,12 +6,13 @@
 
 - css
 - html
+- Javascript
 
 ## Authors
 
 👤 **Author1**
```

Comment:
- [ ] OPTIONAL
Kindly replace the text **Author1** with your full name, so as to make your ReadMe more professional